opflexagent/gbp_ovs_agent.py

```diff
@@ -35,7 +35,6 @@
 from opflexagent import as_metadata_manager
 from opflexagent import constants as ofcst
 from opflexagent import rpc
-from opflexagent import snat_iptables_manager
```

Comment:
Oh this is related to a different changeset. Apparently in Kilo Neutron doesn't pass down the root helper anymore. I'll fix this in master asap